### ISSUE TITLE
Scroll wheel doesn't work

### ISSUE THREAD

**Issue by Erhannis** [JoanSchi/flextable#1]
Testing on Windows, it seems the tables only scroll by clicking and dragging, not by the mouse scroll wheel, which is terribly inconvenient.  Is there a way to get them to respond to the scroll wheel?  I checked the "Tables in ScrollView" example in case that did it, but no dice.

**Comment by JoanSchi** [comment]
Hi @Erhannis ,

I solved in the last version. I need to update some examples and readme/doc.

If you like to try the last version:
```
dependencies:
  flextable:
    git:
      url: https://github.com/JoanSchi/flextable.git
```
Best starting point for a example is other_examples/table_examples. 


If you set TextCell to editable: false and use the textStyle, you have the same functionality with the old version:
```
 TextCell(value: 'test',
    style: TextCellStyle(
      background: color1,
      textStyle:
          TextStyle(fontSize: 20, color: Color.fromARGB(255, 70, 78, 38)),
    ),
editable: false)
```

**Comment by Erhannis** [comment]
Odd, I already cloned the repo and was testing with other_examples/table_examples, so that should be on the most recent commit.  Commit 8f50b770, right?  The scroll wheel doesn't work for me there.

Just in case I misunderstood, I tried changing the "fruit" example so all the TextCell had `editable: false` and a `TextCellStyle` like the one you commented, but the behavior stayed the same.

**Comment by JoanSchi** [comment]
Odd, that one is fine. Did you used windows web or as standalone program?

I tested it on window with flutter web in Chrome for wheel vertical and touchpad vertical and horizontal. It works also on imac in chrome (touchpad scroll).

As long as there is something to scroll it should work.

The event takes place in table_view_scrollable.dart. Around  line 524 in the build there is a listener (onPointerSignal ) that should listenen to the wheel,  probably the listener does not tricker.

Maybe in DeviceGestureSettings mouse is not there, is your platform set to windows?

**Comment by Erhannis** [comment]
Windows standalone - I also just tested with Mac standalone, same non-scrolling behavior.  The scroll wheel does work for a web build, like you say.  I'll look at those lines you mention later today or tomorrow.

**Comment by Erhannis** [comment]
Looks like this is probably the problem?  https://docs.flutter.dev/release/breaking-changes/trackpad-gestures
TL;DR they moved scroll events that originate from a trackpad.  I'm not yet sure how to fix the code, but it looks like they have a lot of example code, at least.

**Comment by JoanSchi** [comment]
Thank you, I will try to fix it next week

**Comment by JoanSchi** [comment]
Hi @Erhannis 

I tested the scroll wheel  for a linux build with flutter stable channel 3.24.3, the scroll wheel still works. If I look at the documentation it should work. 

Unfortunately I can not build for Windows at the moment.  Can you test the scroll wheel with a simple listview on windows?  Thanks.

(I removed the colored cirlces)
